src/containers/Home/Home.scss

```diff
@@ -1,15 +1,36 @@
 .Home{
-    background: rgba(255,255,255,0.9);
-    border-radius: 5px;
+    // background: rgba(255,255,255,0.9);
     width: 30%;
-    margin: auto;
+    color: white;
+    text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
+    // margin: auto;
     display: block;
     padding: 20px;
     h1, button {
         font-family: 'Oswald', sans-serif !important;
     }
     button {
-        width: 80%;
-        margin-bottom: 20px !important;
+        width: 150%;
+        color: white;
+        font-weight: bolder;
+        padding: 0 50px;
+        font-size: 1.5em;
+        height: 75px;
+        position: relative;
+        background-color: transparent !important;
+        margin-bottom: 60px !important;
+        &:focus {
+            outline:0;
+        }
     }
+}
+.Btn_group{
+    margin-left: -20px;
+}
+.Mcgregor{
```

Comment:
The image is getting cut off on my laptop screen. It's height should be responsive instead of the width.

#### Before
![screenshot 23](https://user-images.githubusercontent.com/8203134/35187700-9692fa34-fded-11e7-832b-b3c2accd34dc.png)

#### After
![screenshot 24](https://user-images.githubusercontent.com/8203134/35187699-8ab6f35a-fded-11e7-996b-e5c4c006d936.png)

**CSS change: **
```
{
    position: fixed;
    bottom: 0px;
    right: 30px;
    height: 100%;
    opacity: 1.0;
}
```